code/starfield/starfield.h

```diff
@@ -148,7 +164,9 @@ void stars_preload_background_bitmap(const char *fname);
 
 void stars_set_nebula(bool activate, float range);
 
-void stars_load_debris(int fullneb = 0);
+int get_motion_debris_by_name(SCP_string name);
+
+void stars_load_debris(int fullneb = 0, SCP_string = "");
```

Comment:
The parameters here ought to be `const SCP_string &` to avoid unnecessary copying of the string

Reply:
ok